dev-guide/deploy.md

```diff
@@ -1,13 +1,13 @@
 # Build for deployment
 
-The easiest way to deploy TiDB is to use Docker Compose, see [TiDB Docker Compose Deployment](../dev/how-to/get-started/deploy-tidb-from-docker-compose.md). For more deployment methods, see [TiDB deployment methods](../_index.md).
+For more deployment methods, see [TiDB deployment methods](/_index.md).
```

Comment:
`deploy-tidb-from-docker-compose.md` does not exist in this branch, so I simply removed it. do you have other suggestions?

Reply:
You can also use an absolute link.